app/livechat/client/views/app/tabbar/chatRoomHistoryItem.js

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+import './chatRoomHistoryItem.html';
+import { Template } from 'meteor/templating';
+
+Template.chatRoomHistoryItem.onCreated(function() {
+	const currentData = Template.currentData();
+	currentData.time = moment(currentData.ts).format('LT');
```

Comment:
The `ts` field is not the right value to display in the template, the correct value is the `closedAt` field.
Also, you will need to create a helper method to return this method in the template:

```suggestion
Template. chatRoomHistoryItem.helpers({
	closedAt() {
		const { closedAt } = Template.instance().room.get();
	        return moment(closedAt).format('LT');
	},
```

Reply:
done